### PR TITLE
Added maximum file upload size limit 

### PR DESCRIPTION
**Pull request by ayushin78**
## [DOC-2003](https://openedx.atlassian.net/browse/DOC-2003)

Added maximum file upload size limit in Discussions.
### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### HTML Version (optional)
- [ ] Build an RTD draft for your branch and add a link here
### Sandbox (optional)
- [ ] Point to or build a sandbox for the software change and add a link here
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ayushin78! I've created [OSPR-1451](https://openedx.atlassian.net/browse/OSPR-1451) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: en_us/shared/students/sfd_discussions/add_post.rst [hunk 1]
```diff
@@ -92,10 +92,26 @@ topics on the **Discussion** page.
...
+       The maximum size for an uploaded file is 1 MB.
 
+
+   
+
+   .. only:: Open_edX
+   
+     .. tip::
+       If having trouble with uploading file, please make sure that the 
+       size of file is smaller than 1MB as there might be a maximum size limit.
```
Comment (lamagnifica) [inline en_us/shared/students/sfd_discussions/add_post.rst]: missing space between 1 and MB

**Comment by gsong** [comment]
@ayushin78 Thank you for your PR. We appreciate you taking the time to make the contribution.

Please submit a [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf) to us so we can review this pull request. We realize it's additional work for a simple change in this case, but we're hoping you would make future contributions as well.

**Comment by shaunagm** [comment]
Hi @ayushin78. We're still waiting on the agreement to be processed, but in the meantime, let me direct your attention to the original comment above. You may have noticed that it's a comment attributed to you, but it contains a bunch of info you didn't written. It's called a 'cover letter' and it gets automatically added to pull requests in this repo.

This 'cover letter' provides a useful guide for the steps we need to take to process this PR. One of the main things is review which will be done by someone on the @edx/doc team, but they're going to wait until the agreement gets processed. In the meantime, you can proceed to the next step, testing. Have you run the tests yet? Do they give you any warnings or errors?

Another thing you can do is squash your commits down to their basic changes.  That's something we may need to do again at the end of this process, but right now you have 12 commits when you really only need one.  This is a nice write-up of how to fix this, by rebasing and pushing to your fork: http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit  Please let me know if you need any help with this.  Git can be a difficult tool to learn!

**Comment by ayushin78** [comment]
I tried to squash those commits and apparently, it all got messed.  
So, I have requested a new PR involving one commit only.

**Comment by lamagnifica** [comment]
No worries, @ayushin78 it happens to all of us. What is the new PR number?

**Comment by lamagnifica** [comment]
Closed in favor of https://github.com/edx/edx-documentation/pull/1268

**Pull request closed without merging**